test/client/filters.clienttest.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  'Filtering on search page': function (browser) {
+    browser
+      .url('http://localhost:8000/search?q=charles')
```

Comment:
I find that running the tests on Saucelabs is good for _pass/fail_ but no non-technical person is going to login to sauce, find the test and go through the videos/screenshots ... we need to make it *easy* ...